cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutorContext.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.spi.hbase;
+
+import java.util.Map;
+
+/**
+ * HBase DDL executor context.
+ */
+public interface HBaseDDLExecutorContext {
+  /**
+   * Returns the hadoop conf with which the {@link HBaseDDLExecutor} was initialized.
+   * @param <T>
+   * @return the conf
+   */
+  <T> T getConf();
```

Comment:
let's call this getConfiguration to make it even more clear that it's the Hadoop Configuration class